common/src/main/java/com/thoughtworks/go/serverhealth/HealthStateScope.java

```diff
@@ -232,7 +232,7 @@ public boolean isRemovedFromConfig(CruiseConfig cruiseConfig, String group) {
         },
         MATERIAL {
```

Comment:
This is only done for the material scope. It also needs to be done for `material_update` scope.
As a config repo material can also fail while updating. :)